client/src/js/components/Map/MapContainer.js

```diff
@@ -28,16 +27,10 @@ class Map extends Component {
                 key={foodInstance.id}
                 lat={foodInstance.latitude}
                 lng={foodInstance.longitude}
-                text={foodInstance.name}
+                text={'A'}
```

Comment:
this is probably supposed to be the name right

Reply:
Yeah, I'm going to push something up now. It'll be the corresponding index value to the items in the menu list